KHLY2700/KHLY2700-IOC-01App/Db/keithley2700_channels.template

```diff
@@ -0,0 +1,105 @@
+record(bo, "$(P)$(CHANNEL):SIM")
+{
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(ZNAM, "NO")
+    field(ONAM, "YES")
+    field(VAL, "$(RECSIM=0)")
+}
+
+record(bo, "$(P)$(CHANNEL):DISABLE") 
+{
+    field(DESC, "Disable comms")
+    field(PINI, "YES")
+    field(VAL, "$(DISABLE=0)")
+    field(OMSL, "supervisory")
+    field(ZNAM, "COMMS ENABLED")
+    field(ONAM, "COMMS DISABLED")
+}
+
+##################################################
+#
+# Template for the 20 Keithley_2700 Channels
+#
+# Values created by aSub record BUFFER:READINGS:PARSE which populates channel 
+# values from the block of data returned in BUFFER:READINGS.
+#
+# Channel names range from 101-110 and 201-210 and have the PV format CHNL:101
+#
+##################################################
+
+record(ao, "$(P)$(CHANNEL):READING")
+{
+	field(DESC, "Latest device reading value")
+	field(EGU, "")
+	field(PREC, "5")
+	info(alarm, "Keithley2700")
+	info(INTEREST, "HIGH")
+	info(ARCHIVE, "VAL")
+}
+alias("$(P)$(CHANNEL):READING","$(P)$(CHANNEL):READING:SP")
+
+record(ao, "$(P)$(CHANNEL):TIMESTAMP")
+{
+	field(DESC, "Latest device reading timestamp")
+	field(EGU, "")
+	field(PREC, "5")
+	info(alarm, "Keithley2700")
+	info(INTEREST, "HIGH")
+	info(ARCHIVE, "VAL")
+}
+alias("$(P)$(CHANNEL):TIMESTAMP","$(P)$(CHANNEL):TIMESTAMP:SP")
+
+record(ao, "$(P)$(CHANNEL):TEMP")
+{
+	field(DESC, "Calculated Temperature reading")
+	field(EGU, "K")
+	field(PREC, "5")
+	info(INTEREST, "HIGH")
```

Comment:
Don't have a high interest on this if it hasn't been implemented yet

Reply:
Removed